x-pack/plugins/maps/public/maps_vis_type_alias.js

```diff
@@ -13,7 +13,7 @@ export function getMapsVisTypeAlias(visualizations, showMapVisualizationTypes) {
   }
 
   const description = i18n.translate('xpack.maps.visTypeAlias.description', {
-    defaultMessage: 'Create and style maps with multiple layers and indices.',
+    defaultMessage: 'Create all map visualizations using your geo data.',
```

Comment:
Would it be possible to keep the original text?

Reply:
It was suggested by Raya [here](https://github.com/elastic/kibana/issues/73418#issuecomment-672268662) this is the reason I changed it.

Reply:
I think you can combine the two:

Plot and style your geo data in a multilayer map.

Reply:
Changed it with Gail's suggestion.